docs/reference/commandline/service_rollback.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "service rollback"
```

Comment:
I just read some information about the front-matter.
Should I changed it like that ?
```markdown
---
title: service rollback
description: The service rollback command description and usage
keywords:
 - service
 - rollback
---
```
PS: I just keep the same front-matter used in other command line documentation but I can change it. ^^